app/helpers/task_list_helper.rb

```diff
@@ -3,6 +3,11 @@ def task_status_id(task)
     "#{task.title.parameterize}-status"
   end
 
+  def task_id(task)
```

Comment:
There is a `task_status_id` method in this helper, will that go when the old task list goes?

Reply:
Yep, the why should be in the commit message.

Reply:
Ah, yes. So it is!